build.proj

```diff
@@ -137,6 +137,28 @@
                                     NewVersion="$(NewVersion)" />
   </Target>
 
+  <!-- Packages.zip creation -->
+  <UsingTask TaskName="ZipFileCreateFromDependencyLists" Condition="'$(ArchiveTests)' == 'true'" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
```

Comment:
It is used on the Jenkins x64 machine to generate a zip of the packages needed by the tests. That zip is then transferred to a lab ARM64 machine that unzips it and passes the unzipped directory to 'RunTests.cmd'. I wanted to avoid copying the packages more than once, so I took this bit from the Helix tools.